net-firewall/nftables/nftables-0.9.9.ebuild

```diff
@@ -114,66 +115,3 @@ src_compile() {
 	fi
 }
 
-src_install() {
-	default
-
-	if ! use doc && [[ ! ${PV} =~ ^[9]{4,}$ ]]; then
-		pushd doc >/dev/null || die
-		doman *.?
-		popd >/dev/null || die
-	fi
-
-	local mksuffix="$(usex modern-kernel '-mk' '')"
-
-	exeinto /usr/libexec/${PN}
-	newexe "${FILESDIR}"/libexec/${PN}${mksuffix}.sh ${PN}.sh
-	newconfd "${FILESDIR}"/${PN}${mksuffix}.confd ${PN}
-	newinitd "${FILESDIR}"/${PN}${mksuffix}.init-r1 ${PN}
-	keepdir /var/lib/nftables
-
-	systemd_dounit "${FILESDIR}"/systemd/${PN}-restore.service
-
-	if use python ; then
-		python_foreach_impl python_make install
-		python_foreach_impl python_optimize
-	fi
-
-	find "${ED}" -type f -name "*.la" -delete || die
-}
```

Comment:
I'd probably keep the `src_install()` function, but still reduced to running default and dropping the libtool rubbish. Also, why is `systemd_dounit "${FILESDIR}"/systemd/${PN}-restore.service` dropped too? Do we not need it?

```bash
src_install() {
	default
	systemd_dounit "${FILESDIR}"/systemd/${PN}-restore.service # maybe?
	find "${ED}" -type f -name "*.la" -delete || die
}
```

Reply:
We had the save/restore operations getting dropped in previously as well, so I went ahead and dropped it. 
https://github.com/coreos/coreos-overlay/commit/71d1bd88b493e4e1b2383996d5aa6832aa3e843e